kitchen-tests/kitchen.yml

```diff
@@ -40,7 +40,7 @@ platforms:
   - name: ubuntu-18.04
   - name: ubuntu-20.04
   - name: ubuntu-22.04
-  - name: ubuntu-24.04
+  # - name: ubuntu-24.04
```

Comment:
This is just a configuration item and doesn't need to be commented out. Only the one in the .github workflows file controls what is actually ran .github/workflows/kitchen.yml